includes/class-amp-theme-support.php

```diff
@@ -275,6 +279,44 @@ public static function register_content_embed_handlers() {
 		return $embed_handlers;
 	}
 
+	/**
+	 * Add the comments template placeholder marker
+	 *
+	 * @param array $args the args for the comments list..
+	 * @return array Args to return.
+	 */
+	public static function amp_set_comments_walker( $args ) {
+		$amp_walker     = new AMP_Comment_Walker();
+		$args['walker'] = $amp_walker;
+		// @todo Make threaded work by setting 'data-update-time' of the parent first level parent with the last timestamp.
+		$args['max_depth'] = 1;
```

Comment:
Yeah, we need to be able to remove this constraint.

Reply:
Ye, this is temporary, I figured out how to do it. will experiment and make it work!